### ISSUE TITLE
Cant get it to work. What am i doing wrong?

### ISSUE THREAD

**Issue by nivibilla** [tencent-ailab/IP-Adapter#234]
Hi,

I try to replicate your code on colab but im unable to get any good results, pls help

v1
https://colab.research.google.com/drive/1NrshXd-cBnrKSg_YHsCk-n1VK_XcGmBV?usp=sharing

v2
https://colab.research.google.com/drive/1XkEULw8s-F1eDfehFZ_HujlcQ-1zEwog?usp=sharing

**Comment by xiaohu2015** [comment]
refer to this https://github.com/tencent-ailab/IP-Adapter/issues/189#issuecomment-1867722119

**Comment by nivibilla** [comment]
Ah I see, it doesn't work with safetensors. I will try with the diffusers format.

**Comment by nivibilla** [comment]
Yep works with diffusers version. But hopefully it can be fixed to work with single file too
